client/src/components/mergeReportsDialog/mergeReportsDialog.js

```diff
@@ -0,0 +1,196 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { Modal, Button, Icon } from 'patternfly-react';
+
+import helpers from '../../common/helpers';
+import Store from '../../redux/store';
+import { toastNotificationTypes } from '../../redux/constants';
+import { getMergeScanReults } from '../../redux/actions/reportsActions';
+
+class MergeReportsDialog extends React.Component {
+  constructor() {
+    super();
+
+    helpers.bindMethods(this, ['mergeScanResults']);
+  }
+
+  getValidScans() {
+    const { scans } = this.props;
+    return _.filter(scans, scan => {
+      return _.get(scan, 'most_recent.id') && _.get(scan, 'most_recent.status') === 'completed';
+    });
```

Comment:
Had trouble seeing when `most_recent.id` might not return when combined with the status `completed`?

Reply:
true, if it completed it would have an id. I can simplify.